lib/rubocop/cop/style/redundant_parentheses.rb

```diff
@@ -126,8 +128,17 @@ def only_closing_paren_before_comma?(node)
           line_range.source =~ /^\s*\)\s*,/
         end
 
-        def disallowed_literal?(node)
-          node.literal? && !ALLOWED_LITERALS.include?(node.type)
+        def disallowed_literal?(begin_node, node)
+          node.literal? &&
+            !ALLOWED_LITERALS.include?(node.type) &&
+            !raised_to_power_negative_numeric?(begin_node, node)
+        end
+
+        def raised_to_power_negative_numeric?(begin_node, node)
+          return false unless node.int_type? || node.float_type?
+          return false if node.children.first >= 0 || begin_node.parent.nil?
+
+          begin_node.parent.children[begin_node.sibling_index + 1] == :**
```

Comment:
Same here.

Adding a few locals would certainly improve the readability of your code.